### PR TITLE
[Easy Rules Quartz], Adding quartz module to rules engine.

### PR DESCRIPTION
**Pull request by gs-spadmanabhan**
PR is to add support for quartz scheduler in similar lines to easy batch framework. Let me know your views.

- Sunand

### CODE REVIEW

**Comment by benas** [comment]
Fantastic! Thank you for this addition :+1: 

In fact, I see some use case where we need to apply business rules repeatedly using a scheduler.

I'll merge your PR as soon as possible.

Kind regards
Mahmoud

**Comment by gs-spadmanabhan** [comment]
Hi,

Did you get a chance to take a look?

File: easyrules-quartz/pom.xml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>easyrules</artifactId>
+        <groupId>org.easyrules</groupId>
+        <version>2.0.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quartz</artifactId>
```
Comment (benas) [inline easyrules-quartz/pom.xml]: The artifactId should be "easyrules-quartz". But never mind, I'll update it :-)
The rest is just great! thx.

**Comment by benas** [comment]
Straight to the master branch, thank you for your contribution!

**Comment by gs-spadmanabhan** [comment]
Cool, let me know if you are planning to extend it further like rules chaining(in a way this is already there, like it executes in the order the rules were added to engine), bringing expressions for evaluation etc. I am happy to contribute.

**Pull request merged**